src/APIM_ARMTemplate/apimtemplate/Extractor/Utilities/ExtractorUtils.cs

```diff
@@ -1,436 +0,0 @@
-using apimtemplate.Common.TemplateModels;
-using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common;
-using Microsoft.Extensions.Caching.Memory;
-using Newtonsoft.Json.Linq;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text.RegularExpressions;
-using System.Threading.Tasks;
-
-namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extract
-{
-    static class ExtractorUtils
```

Comment:
What happend to all of this?

Reply:
I moved it to [ExtractorExecutor](https://github.com/DeagleGross/azure-api-management-devops-resource-kit/blob/dmkorolev/607_refactoring/src/APIM_ARMTemplate/apimtemplate/Commands/Executors/ExtractorExecutor.cs), because we can now test this class properly. 

You can pass any `extractor` interface you want for any extractor supported + pass configuration and work with that.
I didn't like the approach of having a single static class for all of that stuff, that generates a lot of `...Extractor` classes for each application launch.